packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -14,11 +14,24 @@ let buildWrapper = template(`
   })
 `);
 
+let arrowBuildWrapper =  template(`
+  (() => {
+    var ref = FUNCTION;
+    return (PARAMS) => ref.apply(this, arguments);
```

Comment:
http://phabricator.babeljs.io/T7006

this should be a regular function.

in node, when i have something like:

```js
const fn = async val => {
  console.log(val)
}
```

Then `fn(val)` will always `console.log(exports)` (from what i remember) because `arguments` is node's `function (exports, module, require, __dirname, __filename) {}` commonjs wrapper